test/defaultResolver.test.js

```diff
@@ -22,49 +21,17 @@ describe('GraphQL - Default Resolver', () => {
 
         // Then
         assert.deepStrictEqual(ret, 'result')
-
-    })
-
-    it('should throw a UserInputError if the uc returns a Err', async () => {
-        // Given
-        const AUseCase = () =>
-            usecase('Use Case X', {
-                request: { id: Number },
-                response: Number,
-                authorize: async () => Ok(),
-                'Step 1': step(() => { return Err("error") }
-                )
-            })
-
-        const resolver = defaultResolver(AUseCase)
-
-        await assert.rejects(
-            // When
-            async () => {
-                await resolver(null, {}, { user: {} })
-            },
-            // Then
-            (err) => {
-                assert.strictEqual(err.name, 'UserInputError')
-                assert.strictEqual(err.extensions.cause, 'error')
-                return true
-            }
-        )
     })
 
-    it('should throw a ForbiddenError if the uc returns a Err.permissionDenied', async () => {
+    it('should not run a use case if not autorized and throw a ForbiddenError', async () => {
         // Given
-        const AUseCase = () =>
-            usecase('Use Case X', {
-                request: { id: Number },
-                response: Number,
-                authorize: async () => Ok(),
-                'Step 1': step(() => {
-                    return Err.permissionDenied({ message: `Permission Denied X`, payload: { entity: 'entity Y' } })
-                })
-            })
+        const AUseCase = {
+            requestSchema: { id: Number },
+            responseSchema: Number,
+            async authorize() { return false }
+        }
 
-        const resolver = defaultResolver(AUseCase)
+        const resolver = defaultResolver(() => AUseCase)
```

Comment:
this is a breaking change?

Reply:
Good question. I will have a look on this.

Reply:
No, it's not. 

Git is just showing this change because I removed a few lines above. I didn't change this line.